### ISSUE TITLE
[Due for payment 2026-02-23] Screen Reader: Workspace Rules: The switch button is announced with an incorrect name

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76906]
## Action Performed:
Prerequisite:
- User is logged in

1. Using JAWS + Chrome, open the URL: https://new.expensify.com/
2. Tab to focus on the required workspace and press Enter to activate it
3. Once the Workspace menu is displayed, tab to focus on the "Distance Rates" menu item and press Enter to activate it
4. Tab to focus on the toggle switch of "Default Rate"
5. Observe the screen reader announcement

## Expected Result:
The switch should be announced with the correct and descriptive name that accurately reflects its visible label and purpose, depending on context, e.g.:
- 'Default Rate, <price>/mile'

## Actual Result:
When navigating to the switch, the screen reader announces it as “Track tax”, which does not accurately describe the control’s actual purpose or visible label. This can confuse users relying on assistive technologies.

HTML:
<button aria-checked="true" aria-label="Track tax" role="switch" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" id="517" style="width: 50px; height: 28px; justify-content: center; border-radius: 20px; padding: 15px; background-color: rgb(3, 212, 124);"><div class="css-g5y9jx" id="518" style="width: 22px; height: 22px; border-radius: 11px; position: absolute; left: 4px; justify-content: center; align-items: center; background-color: rgb(252, 251, 249); transform: translateX(20px);"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="12" height="12" hovered="false" pressed="false" fill="#002E22"><path d="M13.7 8V6c0-2.2-1.8-4-4-4s-4 1.8-4 4v2h-1c-.6 0-1 .4-1 1v6c0 1.7 1.3 3 3 3h6c1.7 0 3-1.3 3-3V9c0-.6-.4-1-1-1zm-2 0V6c0-1.1-.9-2-2-2s-2 .9-2 2v2z" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div></div></button>

## Area issue was found in:
Workspace Rules

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Update the switch's aria-label to correctly match its visible label and accurately describe the control’s actual purpose, e.g.:
- 'Default Rate, <price>/mile'
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-label

## Environment(s)
Windows 11 Home - Chrome

https://github.com/user-attachments/assets/c91c4917-ef5a-4aca-9d01-21f4eda70857

<img width="1918" height="968" alt="Image" src="https://github.com/user-attachments/assets/c0f7fb42-bcc4-4ee4-9840-aba44584dd2c" />

## Screenshots/Videos

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6996751
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
fixed https://github.com/Expensify/App/issues/79233
